internal/plugin.go

```diff
@@ -342,6 +386,46 @@ func (p *Plugin) applyDefaults(unmarshaledConfig map[string]interface{}) {
 				policy.Manifests[i].ComplianceType = policy.ComplianceType
 			}
 		}
+
+		for _, plcsetInPlc := range policy.PolicySets {
+			if _, ok := plcsetToPlc[plcsetInPlc]; !ok {
+				newPlcset := types.PolicySetConfig{
+					Name:     plcsetInPlc,
+					Policies: []string{policy.Name},
+				}
+				p.PolicySets = append(p.PolicySets, newPlcset)
+			}
+			if plcToPlcset[policy.Name] == nil {
+				plcToPlcset[policy.Name] = make(map[string]bool)
+			}
+
+			plcToPlcset[policy.Name][plcsetInPlc] = true
+
+			if plcsetToPlc[plcsetInPlc] == nil {
+				plcsetToPlc[plcsetInPlc] = make(map[string]bool)
```

Comment:
After thinking about this, I think this line could be moved to under `if _, ok := plcsetToPlc[plcsetInPlc]; !ok {`